trax/supervised/configs/bert_mlm_pretraining.gin

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 The Trax Authors.
```

Comment:
together with the other gin file -- these have a lot in common -- can we rename the below gin to bert_nsp_pretraining.gin and extract the common part to bert_pretraning.gin

and include bert_pretraining.gin in both these files?

For an example see https://github.com/google/trax/blob/master/trax/supervised/configs/sparse_c4_pretrain_16gb_adafactor.gin which includes c4.gin - https://github.com/google/trax/blob/master/trax/supervised/configs/c4.gin

Reply:
Thanks! I wasn't aware you can do modularity in gins. I took it step further and extracted bert.gin and refactored all bert-related gins so that they include bert.gin.